toolkit/tools/internal/pkggraph/pkggraph_test.go

```diff
@@ -225,7 +225,27 @@ func buildTestGraphHelper() (g *PkgGraph, err error) {
 	return
 }
 
+// Checks if two lists of nodes are equivalent (ignoring order and graph edges)
+func checkEqualComponents(t *testing.T, expected, actual []*PkgNode) {
+	t.Helper()
+	foundPackage := make(map[*PkgNode]bool)
+	for _, mustHave := range expected {
```

Comment:
Is the `foundPackage` map needed?

Seems like you could just write:

```golang
for _, mustHave := range expected {
	found := false
	for _, n := range actual {
		found = found || mustHave.Equal(n)
	}
	assert.True(t, found, "expected to find %s in actual", mustHave.String())
}
```

Reply:
Fair point, I think this was the remains of a cut-down function with more output before.